src/rest/SearchEndpoint.ts

```diff
@@ -322,14 +322,6 @@ export class SearchEndpoint implements ISearchEndpoint {
     });
   }
 
-  /**
-   * Indicates whether the search endpoint is using JSONP internally to communicate with the Search API.
-   * @returns {boolean} `true` in the search enpoint is using JSONP; `false` otherwise.
-   */
-  public isJsonp(): boolean {
```

Comment:
I *think* that's fine, but an alternative might have been to not modify the interface and just always return false on the implementation (although it is kinda silly...).

I don't recall other integration (eg: Salesforce integration code) relying on that interface but 🤷

Reply:
I was thinking that by removing the method from the interface, it would encourage users of it to adjust their code. I can see not breaking being less burdensome though; I will readd and hard-code to return `false`.